vmprof/stats.py

```diff
@@ -14,10 +14,12 @@ def __init__(self, profiles, adr_dict=None, jit_frames=None, interp=None,
         if state:
             self.profile_lines = state.profile_lines
             self.profile_memory = state.profile_memory
+            self.adr_size = len(state.virtual_ips)
         else:
             # unkown, for tests only
-            self.profile_lines = False
-            self.profile_memory = False
+            self.profile_lines = None
+            self.profile_memory = None
+            self.adr_size = None
```

Comment:
can we change adr_size or remove it? it is simply the len of state.virtual_ips... it should not be confused with the byte size of the addresses contained in the profile. We could turn that into a method with a longer name...